app/shared/components/design-system/all-components/faq/Faq.test.tsx

```diff
@@ -0,0 +1,40 @@
+import '@testing-library/jest-dom';
+import { fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { Faq } from './Faq';
+
+describe('Faq component', () => {
+  const title = 'What is title';
+  const content = 'title is a a sequence of words which defines overall meaning of the text';
+
+  it('renders the Faq title', () => {
```

Comment:
```suggestion
  it('should render the Faq title', () => {
```
And in all tests start writing with 'should'